src/Dev/SapphireTest.php

```diff
@@ -1231,4 +1232,19 @@ protected function mockSleep(int $seconds): DBDatetime
 
         return $now;
     }
+
+    /**
+     * Execute a callback in a given environment type and return the result.
+     */
+    protected function executeInEnvironmentType(string $env, callable $callback): mixed
```

Comment:
```suggestion
    protected function withEnvironmentType(string $env, callable $callback): mixed
```
Matches naming elsewhere for this sort on thing e.g. https://github.com/silverstripe/silverstripe-versioned/blob/2/src/Versioned.php#L2973

You need to update all references to this method in this PR

Reply:
Done